vector/src/main/java/im/vector/home/CommunicateHomeFragment.kt

```diff
@@ -21,39 +21,24 @@ import org.matrix.androidsdk.data.Room
 import org.matrix.androidsdk.data.RoomTag
 import java.lang.IllegalArgumentException
 
-class CommunicateHomeFragment : AbsHomeFragment(), HomeRoomAdapter.OnSelectRoomListener, RegisterListener {
+class CommunicateHomeFragment : BaseCommunicateHomeFragment(), HomeRoomAdapter.OnSelectRoomListener, RegisterListener {
     val dataUpdateListeners = ArrayList<UpDateListener>()
     var result: HomeRoomsViewModel.Result? = null
 
     override fun getLayoutResId(): Int {
         return R.layout.fragment_view_pager_tab
     }
 
-    override fun onCreateView(
```

Comment:
Was onCreateView() accidentally removed as it didn't go into the new BaseCommunicateHomeFragment super class

Reply:
It was my mistake to put that in the first place. That particular piece of code is already handled in Riot's AbsHomeFragment.